lib/cisco_node_utils/vxlan_vtep_vni.rb

```diff
@@ -154,11 +161,23 @@ def remove_add_multicast_group(ip_start, ip_end)
 
     def multicast_group=(range)
       if range == default_multicast_group
-        set_args_keys(state: 'no', ip_start: '', ip_end: '')
-        config_set('vxlan_vtep_vni', 'multicast_group', @set_args)
+        # Due to CSCux78514, trying to remove multicast-group in CLI
+        # when ingress replication is configured results in removing
+        # ingress replication from nvgen. So be careful and negate
+        # Multicast-group only is it is configured.
+        unless multicast_group.empty?
```

Comment:
Ingress_rep can't be configured unless we remove mcast_group. But if we just say no mcast-group even if its not configured, it removes ingress-rep as well which is a defect.